.gitignore

```diff
@@ -28,3 +28,9 @@ dist-ssr
 
```

Comment:
For the commit message : 
🌐(all) add commands to extract & compile translations
Install @formatjs/cli and use it to extract and compile translation strings from
 projects.
I've never seen the word (all), I would suggest (i18n) instead to be more relevant

Reply:
I used `all` with all workspaces in mind (`lib` and `verna-examples-playground`). But indeed, `i18n` is maybe better to explicit the scope.